jupyter-docker/Dockerfile

```diff
@@ -121,16 +114,17 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" \
  && pip install matplotlib-venn \
  && pip install "https://github.com/all-of-us/workbench/archive/pyclient-v0-1-rc3.zip#egg=aou_workbench_client&subdirectory=client/py"
 
-
-ADD spark-defaults.conf.template $SPARK_HOME/conf/
-# Spark defaults conf find and replace [HAILHASH] with the hash
-RUN sed "s|\[HAILHASH\]|$HAILHASH|g" $SPARK_HOME/conf/spark-defaults.conf.template > $SPARK_HOME/conf/spark-defaults.conf
+ADD spark_install_hail.sh $HAIL_HOME/
 ADD jupyter_notebook_config.py $JUPYTER_HOME/
 ADD jupyter_localize_extension.py $JUPYTER_HOME/custom/
-RUN chown -R $USER:users $JUPYTER_HOME
+ADD jupyter_install_notebook_extension.sh $JUPYTER_HOME/
```

Comment:
Previously `jupyter_install_notebook_extension.sh` was being populated through the init bucket which was kind of silly. It's a static utility script which can be added straight to the docker image (along with `spark_install_hail.sh`).

Reply:
👍 I am pro adding more consistency around how resources get into Jupyterland. There's at least three different ways.

Reply:
Agreed. My vague rule of thumb is:
- static files (same for all environments, no templating needed): Dockerfile
- dynamic required files (templated or environment-specific, but always needed like certs): init bucket + cluster-docker-compose.yml
- dynamic optional files (templated or environment-specified, but optional like extension files): init-bucker + docker cp